Source/Applications/SystemCenter/Startup.cs

```diff
@@ -65,7 +66,11 @@ public void Configuration(IAppBuilder app)
 
             HubConfiguration hubConfig = new HubConfiguration();
             HttpConfiguration httpConfig = new HttpConfiguration();
-            
+
+            // Fix serialize issues when fetching serializable models from API
+            httpConfig.Formatters.JsonFormatter.SerializerSettings.ContractResolver =
+            new DefaultContractResolver { IgnoreSerializableAttribute = true };
```

Comment:
What issues with serialization were we having?

Reply:
When retrieving DataFiles from the API, we couldn't parse them because they were returning serialized, causing them not to show up in the tables.

Reply:
This has broader effects than what we need
What endpoint is the issue?

Reply:
`OpenXDA.Types.DataFile` is the issue. I had seen that the Serializable property was added to the model a few years ago and had assumed it was for a good reason. This solution was the only one I found that seemed to work without affecting much else.